spec/support/initializers/rspec_csv_formatter.rb

```diff
@@ -0,0 +1,35 @@
+require "csv"
+
+class CSVFormatter < RSpec::Core::Formatters::JsonFormatter
+  RSpec::Core::Formatters.register self
+
+  def close(_notification)
+    headers = ["Spec", "File", "Status", "Run Time", "Exception"]
+    headers += ["Travis Branch", "Travis URL"]
+    with_headers = { write_headers: true, headers: headers }
+
+    CSV.open(output.path, "w", with_headers) do |csv|
+      @output_hash[:examples].map do |ex|
+        file_path = "#{ex[:file_path]}:#{ex[:line_number]}"
+        row = [ex[:full_description], file_path, ex[:status], ex[:run_time], ex[:exception]]
+        row += [ENV["TRAVIS_BRANCH"], ENV["TRAVIS_BUILD_WEB_URL"]]
+        csv << row
+      end
+    end
+  end
+end
+
+if ENV["TRAVIS"]
+  RSpec.configure do |config|
+    tmp_dir = File.join(Dir.pwd, "tmp")
+    FileUtils.mkdir(tmp_dir) unless File.directory?(tmp_dir)
+
+    csvs_dir = File.join(tmp_dir, "csvs")
+    FileUtils.mkdir(csvs_dir) unless File.directory?(csvs_dir)
+
+    timestamp = DateTime.now.utc.strftime("D%Y-%m-%dT%H-%M-%S")
```

Comment:
Not sure if the leading `D` is on purpose:

```ruby
[1] pry(main)> DateTime.now.utc.strftime("D%Y-%m-%dT%H-%M-%S")
=> "D2020-05-14T08-43-52"
```

my suggestion is to use `Time.current.utc` and add the `Z` at the end, so by looking at files we know the time is always in UTC instead of having to remember down the line at each passage the file will do:

```ruby
Time.current.utc.strftime("%Y-%m-%dT%H-%M-%SZ")
```

will result in:

```ruby
"2020-05-14T08-44-23Z"
```

we could also simply use: 

```ruby
[3] pry(main)> Time.current.utc.iso8601
=> "2020-05-14T08:46:30Z"
```

which is already pretty similar and printable

Reply:
Thanks for the suggestions! I'll do the iso8601 one, gotta stay iso compliant :)